tasks/transform_conformance/babel.snap.md

```diff
@@ -759,21 +759,6 @@ Passed: 269/1081
 * declarations/erased/input.ts
 * declarations/export-declare-enum/input.ts
 * declarations/nested-namespace/input.mjs
-* enum/boolean-value/input.ts
-* enum/const/input.ts
-* enum/constant-folding/input.ts
-* enum/export/input.ts
-* enum/inferred/input.ts
-* enum/inner-references/input.ts
-* enum/mix-references/input.ts
-* enum/non-foldable-constant/input.ts
-* enum/non-scoped/input.ts
-* enum/outer-references/input.ts
-* enum/scoped/input.ts
-* enum/string-value/input.ts
-* enum/string-value-template/input.ts
-* enum/string-values-computed/input.ts
-* enum/ts5.0-const-foldable/input.ts
```

Comment:
I saw you generated the snapshot, but excluded the test in test runner, any reason to do that?

Reply:
We use test cases from TypeScript/Babel as our input source to generate snapshots.
However, our output format differs from TypeScript/Babel, so we have excluded it from the original testing process.

Reply:
ok, got it.